### PR TITLE
Remove references to 3.2.0

### PR DESCRIPTION
**Pull request by samgnaniah**
Removes references to 3.2.0 in the 4.0.0 doc space.

### CODE REVIEW

File: en/docs/reference/customize-product/extending-api-manager/extending-key-management/extending-key-validation.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ public String getJWTValidator() {
 
 ## Extending the KeyValidationHandler Interface
 
-If you are still using reference tokens and you need to extend the key validation similar to the way it was implemented before API-M 3.2.0, you can do so by implementing `org.wso2.carbon.apimgt.keymgt.handlers.KeyValidationHandler`.
+If you are still using reference tokens and you need to extend the key validation similar to the way it was implemented before API-M 4.0.0, you can do so by implementing `org.wso2.carbon.apimgt.keymgt.handlers.KeyValidationHandler`.
```
Comment (nilminiwso2) [inline en/docs/reference/customize-product/extending-api-manager/extending-key-management/extending-key-validation.md:52]: in this case, API-M 3.2.0 should be correct

File: en/docs/reference/accessibility-compliance.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ According to WHO over 2 billion people, about 37.5% of the world's population, h
...
+We at WSO2 are committed to deliver accessible products and build inclusive designs. The **WSO2 APIM 4.0.0 Developer Portal** is now in conformance with Level A and Level AA Success Criteria and techniques of the **Web Content Accessibility Guidelines 2.1** ([WCAG 2.1](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=111%2C123%2C141%2C143%2C144%2C146%2C1412%2C1413%2C211%2C231%2C243%2C244%2C245%2C246%2C247%2C248%2C249%2C2410%2C251%2C252%2C253%2C254%2C311%2C316%2C321%2C322%2C323%2C324%2C325%2C331%2C332%2C411%2C412%2C1411&currentsidebar=%23col_overview&levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl)).
```
Comment (nilminiwso2) [inline en/docs/reference/accessibility-compliance.md:8]:
```suggestion
We at WSO2 are committed to deliver accessible products and build inclusive designs. The **WSO2 APIM 4.0.0 Developer Portal** complies with Level A and Level AA Success Criteria and techniques of the **Web Content Accessibility Guidelines 2.1** ([WCAG 2.1](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=111%2C123%2C141%2C143%2C144%2C146%2C1412%2C1413%2C211%2C231%2C243%2C244%2C245%2C246%2C247%2C248%2C249%2C2410%2C251%2C252%2C253%2C254%2C311%2C316%2C321%2C322%2C323%2C324%2C325%2C331%2C332%2C411%2C412%2C1411&currentsidebar=%23col_overview&levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl)).
```

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-postgresql.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ Follow the  instructions below to set up the PostgreSQL database and users.
     ERROR ApplicationRegistry org.wso2.andes.kernel.AndesException: Connecting to database failed with jndi lookup : WSO2MBStoreDB. data source username : wso2carbon. SQL Error message : General error: java.lang.ArrayIndexOutOfBoundsException
     ```
 
-     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M 3.2.0 pack.
+     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-postgresql.md:81]:
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M distribution.
```

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-oracle.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ Follow the instructions below to set up an Oracle database.
     ERROR ApplicationRegistry org.wso2.andes.kernel.AndesException: Connecting to database failed with jndi lookup : WSO2MBStoreDB. data source username : wso2carbon. SQL Error message : General error: java.lang.ArrayIndexOutOfBoundsException
     ```
 
-     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M 3.2.0 pack.
+     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-oracle.md:72]:
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M distribution.
```

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-oracle-rac.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ Copy the Oracle JDBC libraries (for example, the `<ORACLE_HOME>/jdbc/lib/ojdbc14
     ERROR ApplicationRegistry org.wso2.andes.kernel.AndesException: Connecting to database failed with jndi lookup : WSO2MBStoreDB. data source username : wso2carbon. SQL Error message : General error: java.lang.ArrayIndexOutOfBoundsException
     ```
 
-     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M 3.2.0 pack.
+     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-oracle-rac.md:70]:
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M distribution.
```

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md [hunk 6]
```diff
@@ -145,7 +145,7 @@ Follow the  instructions below to set up a MySQL database:
     ERROR ApplicationRegistry org.wso2.andes.kernel.AndesException: Connecting to database failed with jndi lookup : WSO2MBStoreDB. data source username : wso2carbon. SQL Error message : General error: java.lang.ArrayIndexOutOfBoundsException
     ```
 
-     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M 3.2.0 pack.
+     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md:148]:
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M distribution.
```

File: en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mssql.md [hunk 7]
```diff
@@ -85,7 +85,7 @@ $ pip install mssql-cli
     ERROR ApplicationRegistry org.wso2.andes.kernel.AndesException: Connecting to database failed with jndi lookup : WSO2MBStoreDB. data source username : wso2carbon. SQL Error message : General error: java.lang.ArrayIndexOutOfBoundsException
     ```
 
-     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M 3.2.0 pack.
+     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M pack.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mssql.md:88]:
```suggestion
     1. Replace the MB store database with the default H2 MB store database from a fresh WSO2 API-M distribution.
```

File: en/docs/install-and-setup/setup/api-controller/ci-cd-with-wso2-api-management.md [hunk 8]
```diff
@@ -226,7 +226,7 @@ The **apictl** tool should be installed in the automation servers to begin the p
 
 #### a. Promoting APIs in a Git repository to upper environments via CI/CD
 
-The repository that you committed the project in the above step <a href="#E">E</a> needs to be cloned into the instance that is executing the CI/CD process. From 3.2.0 onwards, **apictl** has the inbuilt support to integrate with a Git based version control system. It gives a unified command `vcs deploy` to deploy any type of project (e.g., APIs, API Products, and Apps).
+The repository that you committed the project in the above step <a href="#E">E</a> needs to be cloned into the instance that is executing the CI/CD process. **apictl** has the inbuilt support to integrate with a Git based version control system. It gives a unified command `vcs deploy` to deploy any type of project (e.g., APIs, API Products, and Apps).
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/api-controller/ci-cd-with-wso2-api-management.md:229]:
```suggestion
The repository that you committed the project in the above step <a href="#E">E</a> needs to be cloned into the instance that is executing the CI/CD process. The API controller (**apictl**) has inbuilt support for integrating with a Git-based version control system. It gives a unified command `vcs deploy` to deploy any type of project (e.g., APIs, API Products, and Apps).
```

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/reference/customize-product/extending-api-manager/extending-key-management/extending-key-validation.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/reference/customize-product/extending-api-manager/extending-key-management/extending-key-validation.md]: Fixed.

**Pull request merged**